File: standard/fizzbuzz.py
for i in range(1,101):
    if i%3 == 0:
        if i%7 == 0:
            print "FizzBuzz"
        else:
            print "Fizz"
    elif i%7 == 0:
        print "Buzz"
    else:
        print str(i)

File: standard/fizzbuzz.rb
i = 0
loop {
  i += 1
  if i % 3 == 0 then
    if i % 5 == 0 then
      puts "FizzBuzz"
    else
      puts 'Fizz'
    end
  else
    if i % 5 == 0 then
      puts 'Buzz'
    else
      puts i
    end
  end

  break if i == 100
}
